### PR TITLE
finish mobile version

### PR DESCRIPTION
**Pull request by kirube1992**
Added work section
Added about section
Added styles for work and about section
Updated README

### CODE REVIEW

**Review by Shisui6 — COMMENTED** [review]
Hi @kirube1992, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a project, that does not have a professional readme. Please make sure to include all sections of the microverse readme [template](https://github.com/microverseinc/readme-template) and personalize them to your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by souad988 — COMMENTED** [review]
Hi @kirube1992 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional readme. Please make sure to include all sections of the microverse readme [template](https://github.com/microverseinc/readme-template) and personalize them to your project
as requested before your README has only the description and built with sections please make sure to add all missing sections from the template.
_The PR title also needs to be changed to something that reflects the work done in this feature : exp: portfolio/setup mobile first ...

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Strangeal — COMMENTED** [review]
Hi @kirube1992, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn’t have a professional `README` and pull request summary. As requested before by the previous reviewer personalize the README to the project needs. Please make sure to change your pull request title as well to reflect the work done in this feature branch: example: portfolio/setup mobile-first... 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi @kirube1992 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- Kindly remove this broken link on your README.
Screenshot 2023-03-25 at 1 25 26 AM
- Since your project name is not Microverse README Template change here with your project name.
Screenshot 2023-03-25 at 1 25 29 AM
- Please remove this template text which is there for informing you not the visitor.
Screenshot 2023-03-25 at 1 24 54 AM
- Please remove this unrelated link on the bottom of your README.
Screenshot 2023-03-25 at 1 29 29 AM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @kirube1992,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

https://github.com/kirube1992/new-portfolio/blob/3f62701d79d620bdb55db5a28dff7258c58d138f/index.html#L13-L152

- [ ] It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. Currently, you have multiple `h1` tags on your page. Kindly make it one. :+1: 

- [ ] Currently, you have used a ul for all elements. Kindly remove it as the ul tag is used to list items. :+1: 

- [ ] The main container of the projects has a display: grid. Please work on it :+1: 

- [ ] Your design is not matched the Figma design. Kindly work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_work | figma_work

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_myself | figma_myself


https://github.com/kirube1992/new-portfolio/blob/3f62701d79d620bdb55db5a28dff7258c58d138f/index.html#L127-L150

- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li. :+1: 

- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. :+1: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by kirube1992** [comment]
I correct all suggestion if anything else be more detail

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @kirube1992 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
- [ ] Please make sure your work section looks exactly like the FIgma template in terms of (margin, padding, border, button hover effect, and color) because what you have currently doesn't look the same. 

Your design:
Screenshot 2023-03-25 at 01 28 48


Figma Design: 
Screenshot 2023-03-25 at 01 32 17

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -44,73 +43,82 @@ <h1 class="item1 h1">I’m Kirubel. <br>Glad to see you!</h1>
     </ul>
   </section>
   <section class="work1">
-    <img class="work1-img" src="./images/Snapshoot Portfolio (1).png" alt="">
-    <h1>Tonic</h1>
+    <img class="work1-img" src="./images/port.png" alt="snapshot of website">
+    <p class="oldo">Tonic</p>
     <ul>
```
Comment (Qoosim) [inline index.html:51]: - [ ] As mentioned by the previous reviewer, the margin between the headline and your works section is not as we have it on the Figma design. Kindly look into it and fix.

### Your Design
your-header-projects

### Figma Design
temp-header-project

File: index.html [hunk 2]
```diff
@@ -126,27 +134,27 @@ <h1 class="item1 h1">About Myself</h1>
...
           </div>
           <div class="abfr">
             <p>FrameWork</p>
-            <img class="abfr1" src="./images/Disabled.png" alt="">
+            <img class="abfr1" src="./images/download.png" alt="arrow icon">
           </div>
           <div class="abfr">
             <p>Skill</p>
-            <img class="abfr1" src="./images/Disabled.png" alt="">
+            <img class="abfr1" src="./images/download.png" alt="arrow icon">
```
Comment (Qoosim) [inline index.html]: - [ ] Great job implmenting the changes from previous reviewer. However, there is one small thing you need to work on. The direction of the arrow on the language and the framework and skill are not align with what we have on the Figma design. Kindly look at them and fix.

### Your Design
your-lang

### Figma Design
temp-lang

File: index.html [hunk 3]
```diff
@@ -44,73 +43,82 @@ <h1 class="item1 h1">I’m Kirubel. <br>Glad to see you!</h1>
...
+       <ul>
        <div class="work1-link">
-      <li>HTML</li>
-      <li>CSS</li>
-      <li>JavaScript</li>
+      <li >HTML</li>
+      <li class="newlin">CSS</li>
+      <li class="newlin">JavaScript</li>
       </div>
+    </ul>
```
Comment (Qoosim) [inline index.html:83]: - [ ] As suggested by the previous reviewer, I would like to recommend you to give a closer look into the figma design and make your design look almost exactly. This includes, color, font-size, font-weight, font-style, margin, padding background-color among others. I would love to encourage your to work your design alongsize the figma design element by element.

### Your Design
your-works

### figma Design
temp-works

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @kirube1992, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] The link to the MIT License is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up.

license


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @kirube1992, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes completed :heavy_check_mark: 

## Optional suggestions
- I will suggest that you improve upon the mobile view of your project. Even though you fixed all the changes requested, you can see that your project is still missing some aspects of the original FIgma design. So I will suggest that in your free time, you try to work on that so your project looks exactly like the Figma Design :+1: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**